Rubberduck.CodeAnalysis/Inspections/Concrete/MissingAttributeInspection.cs

```diff
@@ -1,87 +1,59 @@
 using System.Collections.Generic;
 using System.Linq;
-using Antlr4.Runtime;
 using Rubberduck.Inspections.Abstract;
 using Rubberduck.Inspections.Results;
 using Rubberduck.Parsing;
 using Rubberduck.Parsing.Annotations;
-using Rubberduck.Parsing.Grammar;
 using Rubberduck.Parsing.Inspections;
 using Rubberduck.Parsing.Inspections.Abstract;
 using Rubberduck.Parsing.Symbols;
 using Rubberduck.Parsing.VBA;
-using Rubberduck.Parsing.VBA.Parsing;
 using Rubberduck.Resources.Inspections;
 
 namespace Rubberduck.Inspections.Concrete
 {
     [CannotAnnotate]
-    public sealed class MissingAttributeInspection : ParseTreeInspectionBase
+    public sealed class MissingAttributeInspection : InspectionBase
     {
         public MissingAttributeInspection(RubberduckParserState state)
             : base(state)
-        {
-            Listener = new MissingMemberAttributeListener(state);
-        }
-
-        public override CodeKind TargetKindOfCode => CodeKind.AttributesCode;
-
-        public override IInspectionListener Listener { get; }
+        {}
 
         protected override IEnumerable<IInspectionResult> DoGetInspectionResults()
         {
-            return Listener.Contexts.Select(context =>
-            {
-                var name = string.Format(InspectionResults.MissingAttributeInspection, context.MemberName.MemberName,
-                    ((VBAParser.AnnotationContext)context.Context).annotationName().GetText());
-                return new QualifiedContextInspectionResult(this, name, context);
-            });
-        }
-
-        public class MissingMemberAttributeListener : ParseTreeListeners.AttributeAnnotationListener
-        {
-            public MissingMemberAttributeListener(RubberduckParserState state) : base(state) { }
-
-            public override void ExitAnnotation(VBAParser.AnnotationContext context)
+            var declarationsWithAttributeAnnotations = State.DeclarationFinder.AllUserDeclarations
```

Comment:
Can this inspection be ignored? If so, I can't find where that's being filtered. If not, cool.

Reply:
It can't. See the attribute on top.
Honestly, it does not make much sense to put an ignore this when you can just remove the annotation instead.